docs/source/user/file_formats.md

```diff
@@ -1,162 +1,151 @@
+JupyterLab provides a unified architecture for viewing and editing data in a
+wide variety of formats. This model applies whether the data is in a file or is
+provided by a kernel as rich cell output in a notebook or code console.
 
 For files, the data format is detected by the extension of the file. A single
 file extension may have multiple editors or viewers registered. For example a
-Markdown file (`.md`) can be edited in the File Editor, or rendered inline.
-
-You can access different editors and viewers for a file, by right-clicking on
-the filename in the File Browser and using the "Open With..." submenu:
+Markdown file (`.md`) can be edited in the file editor or rendered and displayed
+as HTML. You can open different editors and viewers for a file by right-clicking
+on the filename in the file browser and using the "Open With..." submenu:
 
 [screenshot]
 
-To use these different data formats as output in the Notebook or Code Console,
-you can use the relevant display API for the kernel you are using. For example,
-the IPython kernel provides a `display` function that takes a `dict` of keys
-(MIME types) and value (MIME data):
+To use these different data formats as output in a notebook or code console, you
+can use the relevant display API for the kernel you are using. For example, the
+IPython kernel provides a variety of convenience classes for displaying rich output:
 
 ```python
-from IPython.display import display
-display({'text/html': '<h1>Hello World</h1>'}, raw=True)
+from IPython.display import display, HTML
+display(HTML('<h1>Hello World</h1>'))
 ```
 
-Running this code will display the HTML in the output of the notebook or code
-console:
+Running this code will display the HTML in the output of a notebook or code
+console cell:
 
 [screenshot]
 
-Other Jupyter Kernels offer similar APIs.
+The IPython display function can also construct a raw rich output message from a
+dictionary of keys (MIME types) and values (MIME data):
+
+```python
+from IPython.display import display
+display({'text/html': '<h1>Hello World</h1>', 'text/plain': 'Hello World'}, raw=True)
+```
+
+Other Jupyter kernels offer similar APIs.
+
+The rest of this section highlights some of the common data formats that
+JupyterLab supports.
 
 ## Markdown
 
-* File format: `.md`
+* File extension: `.md`
 * MIME type: `text/markdown`
 
 Markdown is a simple and popular markup language used for text cells in the
 Jupyter Notebook.
 
 Markdown documents can be edited as text files or rendered inline:
 
-[animation]
+[animation showing opening a markdown document editor and renderer side-by-side, and changes in the editor being reflected in the renderer]
 
-The markdown syntax supported in this mode is the same as that in the Jupyter
-Notebook (LaTeX equations work). As seen in the animation, edits to the markdown
-source are immediately reflected in the rendered version:
+The Markdown syntax supported in this mode is the same syntax used in the
```

Comment:
Should we provide additional links or references here to the markdown syntax we accept?

Reply:
I don't think we have a formal spec for our markdown syntax. That's an ongoing conversation.